types/src/dkg/real_dkg/mod.rs

```diff
@@ -204,14 +265,38 @@ impl DKGTrait for RealDKG {
             .map(|address| (params.pvss_config.epoch, address))
             .collect::<Vec<_>>();
 
-        trx.verify(
+        trx.main.verify(
             &params.pvss_config.wconfig,
             &params.pvss_config.pp,
             &spks,
             &all_eks,
             &aux,
         )?;
 
+        // Verify fast path is present if and only if fast_wconfig is present.
+        ensure!(
+            trx.fast.is_some() == params.pvss_config.fast_wconfig.is_some(),
+            "real_dkg::verify_transcript failed with mismatched fast path flag in trx and params."
+        );
+
+        if let Some(fast_trx) = trx.fast.as_ref() {
+            let _fast_dealers = fast_trx
```

Comment:
why prefix with _?

Reply:
Lint complained about it. Will try again and see.